src/node/NodeOptions.hs

```diff
@@ -79,120 +83,143 @@ argsParser = do
         long    "db-path" <>
         metavar "FILEPATH" <>
         value   "node-db" <>
-        help    "Path to the node database"
+        help    "Path to directory with node’s database (called Global State). \
```

Comment:
No, it's not part to Global State. Global State in only one of DBs. This directory contains all DBs used by node.